buildpipeline/pipelines.json

```diff
@@ -352,6 +391,11 @@
             "GitHubRepositoryName": "coreclr",
             "AzureContainerPackageGlob": "pkg\\*.nupkg",
             "AzureContainerSymbolPackageGlob": "symbolpkg\\*.nupkg"
+          },
+          "ReportingParameters": {
+            "SubType":  "Publish",
+            "Type": "build/publish/",
+            "PB_BuildType": "Release"
```

Comment:
Did that earlier today - https://github.com/dotnet/core-eng/commit/f44d5e6e08470cd45ecb30136673d400b6901126